app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<html>
+<body>
+
+<div class="flex-centered bg-primary ctr-primary ">
+<div class=" flex-column rad-10 w-30">
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  
+  <div class="form-field flex-column-centered mt-m10">
+
+    <div class="field flex-start">
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "input", placeholder: "Email" %>
+    </div>
+
+    <div class="name-field flex-start">
+      <%= f.text_field :name , class: "input", placeholder: "Full name" %>
+    </div>
+
+    <div class="field flex-start">
+      <%= f.password_field :password, autocomplete: "new-password", class: "input", placeholder: "Password"  %>
+    </div>
+
+    <div class="field flex-start">
+      <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "input", placeholder: "Confirm Password"   %>
+    </div>
+    <div class="actions flex-centered ">
+      <%= f.submit "Register" %>
+    </div>
+  <% end %>
+  
+  <div class="flex-start">
+  <%= render "devise/shared/links" %>
+  </div>
```

Comment:
- [ ] @kalbek great work on your registration page, however, it requires a few adjustments to match the design given, below is a comparison and highlight of sections that need adjustings


Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/kalbek/Budget-app/assets/61408860/a6914bcf-88fe-4e37-afcd-d40aa5c58a87)  | ![image](https://github.com/kalbek/Budget-app/assets/61408860/2cbdd2ee-8bb7-434e-9f31-32b6db59722a)